vladiate/validators.py

```diff
@@ -208,3 +210,26 @@ def validate(self, field, row={}):
     @property
     def bad(self):
         pass
+
+
+def stringify_set(a_set, max_len, max_sort_size=8192):
+    ''' Stringify `max_len` elements of `a_set` and count the remainings
+
+    Small sets (len(a_set) <= max_sort_size) are displayed sorted.
+    Large sets won't be sorted for performance reasons.
+    This may result in an arbitrary ordering in the returned string.
+    '''
+    if max_len < 0:
+        raise ValueError("max_len must be non-negative: {}".format(max_len))
+    if max_sort_size < 0:
+        raise ValueError(
+            "max_sort_size must be non-negative: {}".format(max_sort_size))
```

Comment:
I appreciate these checks but since this is just a utility function for use and doesn't take user input, I think they're unnecessary. We can just drop d355def entirely.